src/ResourceManager/Intune/Commands.Intune/ApiVersionHandler.cs

```diff
@@ -0,0 +1,95 @@
+﻿// ----------------------------------------------------------------------------------
```

Comment:
does this mean we cant expose apiVersion param in cmdlets? We differentiate the environments by tag change -beta,-preview etc..